apps/extension/src/ui/apps/popup/pages/Portfolio/PortfolioAccounts.tsx

```diff
@@ -364,18 +312,22 @@ export const PortfolioAccounts = () => {
     [balancesByAddress, currency, folder]
   )
 
+  const hasFunds = useMemo(() => balances.sum.planck.total > 0n, [balances])
+  const showGetStartedPopup = !hasFunds && ownedAccounts.length <= 2
```

Comment:
balances aren't suspensed so this will flicker, sorry i said the contrary during the meeting
might want to use accounts (which will be suspensed with metadata PR) + the hasFunds flag that we have in appState (also suspensed)